releasenotes/releasenotes_V0.9.0.md

```diff
@@ -1,4 +1,4 @@
-# Release Notes develop
+# Release Notes 0.9.0
 
```

Comment:
Why are the release notes empty?

Reply:
they are not

Reply:
https://github.com/keptn-contrib/dynatrace-sli-service/blob/cc4b54255c4f03dd3ddf426759405587d507cae6/releasenotes/releasenotes_V0.9.0.md

Reply:
Sorry, I was too fast.